plugins/dell-dock2/fu-dell-dock2-common.c

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2024 Dell Technologies
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR MIT
+ */
+
+#include "config.h"
+
+#include "fu-dell-dock2-common.h"
+
+gchar *
+fu_hex_version_from_uint32(guint32 val, FwupdVersionFormat kind)
```

Comment:
I think this should go to `fu-version-common.h`

Reply:
Fixed